src/mscorlib/src/System/Collections/Generic/Dictionary.cs

```diff
@@ -129,6 +129,24 @@ private struct Entry {
             }
         }
 
+        public Dictionary(IEnumerable<KeyValuePair<TKey, TValue>> enumerable):
+            this(enumerable, EqualityComparer<TKey>.Default) { }
+
+        public Dictionary(IEnumerable<KeyValuePair<TKey, TValue>> enumerable, IEqualityComparer<TKey> comparer):
+            this(comparer)
+        {
+            if( enumerable == null) {
```

Comment:
Nit: There should be a space between the `if` and `(`, and the space between `(` and `enumerable` should be deleted.